### DISCUSSION TITLE
Adding kafka ACL support is on your road map

### DISCUSSION THREAD

**Discussion by ramamanoj142000** [provectus/kafka-ui#1649] [Q&A] [answered]
Hi ,
Are you planning to Add Kafka ACL on Kafka -ui in the future anytime. we are looking at that feature. 
I would like to contribute in this, what is the best way to start on contributing on code fix or enhancement.

**Comment by Haarolean** [comment, marked as answer]
Hi, do you mean managing Kafka's ACLs? In this case, we do plan this (issue #124).
Our contributing guide is located [here](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md), there are also instructions on how to pick an issue.
I'm not sure if #124 is a good candidate for a new-coming contributor and we didn't even take a look in it yet. But if you're familiar with java and/or typescript and kafka, you could try digging into it.

**Reply by ramamanoj142000** [reply]
do u have any design document on this source code  e.g. framework  
How to run locally if i want to run

**Reply by Haarolean** [reply]
We have instructions on running locally described in our contributing guideline.
Regarding the frontend frameworks -- we better ask @workshur @Motherships
